indico/web/errors.py

```diff
@@ -46,6 +48,19 @@ def load_error_data(uuid):
     return error_cache.get(uuid)
 
 
+def _handle_error_cors():
+    """Set CORS on error response based on RH's CORS settings."""
+    if not hasattr(g, 'rh') or (cors_config := getattr(g.rh, '_CORS', False)) is False:
```

Comment:
```suggestion
    if not hasattr(g, 'rh') or (cors_config := getattr(g.rh, '_CORS', None)) is None:
```

forgot to change this...